contribute/style-guides/documentation-style-guide.md

```diff
@@ -53,6 +53,8 @@ The following sections provide general guidelines on topics specific to Grafana
 - Use active voice. Avoid passive voice.
   - Active: Grafana displays the heatmap visualization.
   - Passive: The heatmap visualization is displayed.
+  - Active: After you create a dashboard, you can add a panel to it.
```

Comment:
I recommend a new subsection:

Write directly to the reader.
- Use:    "After you create a dashboard, you can add a panel to it."
- Avoid: "After you create a dashboard, it is possible to add a panel to it."

Alternatively, change line 57 to, "After a dashboard has been created, it is possible to add a panel to it."